packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts.scss

```diff
@@ -70,8 +70,12 @@
             width: 15rem;
             height: 2rem;
             background: var(--status-transfer);
-            color: var(--text-colored-background);
+            color: var(--text-colored-background) !important;
```

Comment:
Why need this `!important` ?

Reply:
The background color was not getting applied. I had to use important. Do I need to find the root of the issue or this change suffices ?

Reply:
Then we need to tweak the main class inside this css style like `.dc-modal` something